docs/platforms/jfrog.md

```diff
@@ -41,33 +41,35 @@ terraform {
   required_providers {
     coder = {
       source  = "coder/coder"
-      version = "~> 0.11.1"
     }
     docker = {
       source  = "kreuzwerker/docker"
-      version = "~> 3.0.1"
     }
     artifactory = {
       source  = "registry.terraform.io/jfrog/artifactory"
-      version = "~> 8.4.0"
     }
   }
 }
 
-variable "jfrog_host" {
+variable "jfrog_url" {
```

Comment:
won't this break existing deployments?

Reply:
Nope. As our templates are not auto updated. 
This is just changing the example template.